src/LinkedIn.php

```diff
@@ -335,6 +336,15 @@ protected function getUrlGenerator()
         return $this->urlGenerator;
     }
 
+    /**
+     * @param RequestManagerInterface $manager
+     */
+    public function setRequestManager(RequestManagerInterface $manager) {
```

Comment:
Im not okey with adding these public methods just to make it easier with the tests.

Reply:
OK; I've changed them to be constructor parameters instead.